test/terraform_aws_s3_example_test.go

```diff
@@ -36,6 +36,8 @@ func TestTerraformAwsS3Example(t *testing.T) {
 			"tag_bucket_name":        expectedName,
 			"tag_bucket_environment": expectedEnvironment,
 			"with_policy":            "true",
+			// Passing region to Terraform here is required for the terraform validate command to pass
+			"region": awsRegion,
```

Comment:
If we're now setting the region this way, we should remove the `AWS_DEFAULT_REGION` env var configuration down below.

Reply:
Fixed in f5c6a3f